.github/workflows/pytest.yml

```diff
@@ -28,15 +28,15 @@ jobs:
           - python-version: "3.8"
             pytorch-version: 1.13.0
             torchscript-version: 1.10.2
-            ray-version: 2.0.0
+            ray-version: 2.1.0
```

Comment:
We should probably increase the minimal Ray version in `requirements_distributed.txt` and `requirements_hyperopt.txt` to 2.1 to be safe. Altneratively, I would keep the min tested version as 2.0 here and not bother testing 2.1 (since we test 2.2 and 2.3).

Reply:
@tgaddair Going to do 2.0, 2.2 and 2.3